packages/jest-transform/src/shouldInstrument.ts

```diff
@@ -81,7 +81,9 @@ export default function shouldInstrument(
   }
 
   if (
-    config.coveragePathIgnorePatterns.some(pattern => !!filename.match(pattern))
+    config.coveragePathIgnorePatterns.some(pattern =>
```

Comment:
Is something putting stuff onto the next line maybe?
these changes do not seem related to the rule itself
I don't mind it, but am wondering if you are using some autofix for this